web/static/httprest/connect/styles.css

```diff
@@ -0,0 +1,142 @@
+body {
```

Comment:
There's a top level `web/static/styles.css` with a lot of common content.

Reply:
Notice that different integrations have subtle differences in their "style.css" files. This is intentional - future integrations may not be part of the AK core, and their UIs will be served independently, so they shouldn't rely on AK's `/styles.css` file either.